datasource/auth.go

```diff
@@ -22,7 +22,8 @@ import (
 	"github.com/apache/servicecomb-service-center/pkg/rbacframe"
 )
 
-type Auth interface {
+// AccountManager contains the RBAC CRUD
+type AccountManager interface {
```

Comment:
change the file name from auth ao account?

Reply:
DONE